book/src/validator-stake.md

```diff
@@ -13,12 +13,29 @@ and use the wallet's `delegate-stake` command to stake your validator with 42 la
 $ solana-wallet delegate-stake ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json 42
 ```
 
-Note that stake changes are applied at Epoch boundaries so it can take an hour
-or more for the change to take effect.
+Note that stakes need to warm up, and warmup increments are applied at Epoch boundaries, so it can take an hour
+or more for the change to fully take effect.
 
-Stake can be deactivate by running:
+Assuming your node is voting, now you're up and running and generating validator rewards.  You'll want
+to periodically redeem/claim your rewards:
+
+```bash
+$ solana-wallet redeem-vote-credits ~/validator-config/stake-keypair.json ~/validator-vote-keypair.json
+```
+
+The rewards lamports earned are split between your stake account and your vote account according to the
```

Comment:
"your stake account and your vote account" -- It could be my stake and your vote account too, or even multiple stake accounts.

Reply:
just following the pattern above these lines, but have reduced "your" to stake, which is implied by the audience having the keypair